### PR TITLE
fix: push plan subscription failure command not created for JDBC

### PR DESCRIPTION
**Pull request by Okhelifi**
## Issue

https://gravitee.atlassian.net/browse/APIM-6600

## Description

This PR follow this one https://github.com/gravitee-io/gravitee-api-management/pull/9042 it's simply a cherry pick of the commit and we use a different versions for the webhook dependency.

In 4.4.x we have introduced the environmentId in the subscription that's why here I map it in the command as it can be a usefull data which become accessible in this version 

Linked PRs:
- https://github.com/gravitee-io/gravitee-api-management/pull/9042
- https://github.com/gravitee-io/gravitee-entrypoint-webhook/pull/62
- https://github.com/gravitee-io/gravitee-gateway-api/pull/236 ( for 4.4.x )
- https://github.com/gravitee-io/gravitee-gateway-api/pull/235 ( for master )

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-reduicavej.chromatic.com)

### CODE REVIEW

File: gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.1.25
+      author: GraviteeSource Team
```
Comment (graviteeio) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml:4]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]

Comment (graviteeio) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml:4]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]

File: pom.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
         <gravitee-exchange.version>1.8.2</gravitee-exchange.version>
         <gravitee-expression-language.version>3.2.1</gravitee-expression-language.version>
         <gravitee-fetcher-api.version>2.0.0</gravitee-fetcher-api.version>
-        <gravitee-gateway-api.version>3.5.0</gravitee-gateway-api.version>
+        <gravitee-gateway-api.version>3.5.1-apim-6600-add-envid-in-subscription-350-SNAPSHOT</gravitee-gateway-api.version>
```
Comment (phiz71) [inline pom.xml]: Don't forget to replace with the released version.

File: pom.xml [hunk 3]
```diff
@@ -274,7 +274,7 @@
         <gravitee-entrypoint-http-get.version>1.2.0</gravitee-entrypoint-http-get.version>
         <gravitee-entrypoint-http-post.version>1.2.0</gravitee-entrypoint-http-post.version>
         <gravitee-entrypoint-sse.version>4.1.0</gravitee-entrypoint-sse.version>
-        <gravitee-entrypoint-webhook.version>3.0.0</gravitee-entrypoint-webhook.version>
+        <gravitee-entrypoint-webhook.version>3.0.1-apim-6600-handle-malformed-messages-SNAPSHOT</gravitee-entrypoint-webhook.version>
```
Comment (phiz71) [inline pom.xml]: Don't forget to replace with the released version

**Review by phiz71 — CHANGES_REQUESTED** [review]

**Comment by phiz71** [comment]
Reminder: use the right version of gravitee-api on mergify PRs for 4.5 and master

File: gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml (hunk 1 above)
Comment (graviteeio) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml:4]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]

File: pom.xml (hunk 2 above)
Comment (Okhelifi) [reply pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (Okhelifi) [reply pom.xml]: Done

File: gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml (hunk 1 above)
Comment (graviteeio) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml:4]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]

Comment (graviteeio) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_1_25/schema.yml:4]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**